scripts/run_script_e2e.sh

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
+
+shout() {
+  set +x
+  echo -e "\n!!!!!!!!!!!!!!!!!!!!\n${1}\n!!!!!!!!!!!!!!!!!!!!\n"
+  set -x
+}
+
+# Prep for integration/e2e
+shout "Building odo binaries"
+make bin
+
+# copy built odo to GOBIN
+cp -avf ./odo $GOBIN_TEMP/
+
+# Integration tests
+shout "Testing against 4x cluster"
+
+shout "Logging into 4x cluster for some setup (logs hidden)"
+shout $CLUSTER_TYPE
+set +x
+if [[ $CLUSTER_TYPE == "PSI" ]]
+then
+    #PSI cluster login
+    oc login -u kubeadmin -p ${OCP4X_KUBEADMIN_PASSWORD} --insecure-skip-tls-verify  ${OCP4X_API_URL}
+else
+    # Login to IBM Cloud using service account API Key
+    ##ibmcloud login --apikey $IBMC_OCP47_APIKEY -a cloud.ibm.com -r eu-de -g "Developer CI and QE"
+
+    # Login to cluster in IBM Cloud using cluster API key
+    ##oc login --token=$IBMC_OCLOGIN_APIKEY --server=$IBMC_OCP47_SERVER
+    bash /Users/jbossqa/odo/envset.sh
+fi
+set -x
+
+shout "Doing some presetup"
+
+# Delete any projects with SCRIPT_IDENTITY PREFIX. This is GC from previous runs which fail before end of script cleanup
+for i in $(oc projects -q); do
+    if [[ $i == "${SCRIPT_IDENTITY}"* ]]; then
+        oc delete project $i
+    fi
+done
+
+# Generate random project names to some tests
+export REDHAT_OPENJDK11_RHEL8_PROJECT="${SCRIPT_IDENTITY}$(cat /dev/urandom | tr -dc 'a-z0-9' | fold -w 4 | head -n 1)"
+export REDHAT_OPENJDK11_UBI8_PROJECT="${SCRIPT_IDENTITY}$(cat /dev/urandom | tr -dc 'a-z0-9' | fold -w 4 | head -n 1)"
+export REDHAT_NODEJS12_RHEL7_PROJECT="${SCRIPT_IDENTITY}$(cat /dev/urandom | tr -dc 'a-z0-9' | fold -w 4 | head -n 1)"
+export REDHAT_NODEJS12_UBI8_PROJECT="${SCRIPT_IDENTITY}$(cat /dev/urandom | tr -dc 'a-z0-9' | fold -w 4 | head -n 1)"
+export REDHAT_NODEJS14_UBI8_PROJECT="${SCRIPT_IDENTITY}$(cat /dev/urandom | tr -dc 'a-z0-9' | fold -w 4 | head -n 1)"
+
```

Comment:
Why are we not handling operator namespace here https://github.com/openshift/odo/blob/main/scripts/configure-installer-tests-cluster.sh#L23

Reply:
As configuring cluster was one time thing, but as we are changing it to gingko taking care of installing we will remove this from the script.
As installing operator is for not only a one time setup we have not added it to the setup/run script